.github/workflows/beta-build.yml

```diff
@@ -3,46 +3,61 @@ on:
   push:
     branches:
       - beta
+    # only run on changes to these files
+    paths:
+      - "**.js"
+      - "**.jsx"
 
 # Publishes beta builds with the dist tag of "beta" and
-# with a version of 0.0.0 with a short git SHA appended to
-# the end.
 jobs:
   publish:
     name: Beta block publish
     runs-on: ubuntu-latest
     env:
+      NODE_AUTH_TOKEN: ${{ secrets.NODE_AUTH_TOKEN }}
       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
     steps:
+      - name: Set git username
+        run: git config --global user.email "jack.howard@washpost.com" && git config --global user.name "Jack Howard"
```

Comment:
We shouldn't really rely on specific developers accounts here, I've created a ticket to address this more globally - https://arcpublishing.atlassian.net/browse/TMEDIA-382

Reply:
that's a good point. @RobinGiannattasio talked about using a recently-created bot account that I think engine team is using, right? @badmintonkid

Reply:
you'll need a user with publishing access. engine's is a dummy that doesn't have access to anything just so we can access public github packages. but you can do the same process to set one up. luke created a email distribution that we used for the email on the account signup.

Reply:
ah right thanks @badmintonkid